client/client_pool/src/concord_client_pool.cpp

```diff
@@ -414,12 +414,12 @@ void SingleRequestProcessingJob::execute() {
     read_config_.request.correlation_id = correlation_id_;
     read_config_.request.span_context = span_context_;
     res = processing_client_->SendRequest(read_config_, std::move(request_));
-    reply_size = res.matched_data.size();
+    reply_size = res.matched_data.size() + sizeof(res.result);
     if (callback_) {
-      if (processing_client_->getClientRequestError() != TIMEOUT) {
+      if (processing_client_->getRequestExecutionResult() != TIMEOUT) {
         callback_(bftEngine::SendResult{res});
       } else {
-        callback_(bftEngine::SendResult{SubmitResult::TimedOut});
```

Comment:
Remove irrelevant values from SubmitResult

Reply:
Removed timeout. We cannot remove Invalid argument as its being used .
```
SubmitResult ConcordClientPool::SendRequest(const bft::client::ReadConfig &config,
                                            bft::client::Msg &&request,
                                            const bftEngine::RequestCallBack &callback) {
  LOG_INFO(logger_, "Received read request with cid=" << config.request.correlation_id);
  if (callback && config.request.pre_execute) {
    callback(bftEngine::SendResult{SubmitResult::InvalidArgument});
```